<title>Issue 6347 -   chromium -    test_shell_test memory corruption on mac dbg buildbot - Project Hosting on Google Code</title> <pre> It's probably temporary and intermittent  but may be a symptom of more  serious problems.    Log link:  <a href="http://build.chromium.org/buildbot/waterfall/builders/Modules%20Mac%20%28dbg%29/builds/3529/steps/test_shell_tests/logs/stdio">http://build.chromium.org/buildbot/waterfall/builders/Modules%20Mac%20%28dbg%29/builds/3529/steps/test_shell_tests/logs/stdio</a>    Snip:    [ RUN      ] ResourceFetcherTests.ResourceFetcherTimeout  /b/pylibs/zope/__init__.py:19: UserWarning: Module zope was already  imported from /b/pylibs/zope/__init__.pyc  but  /System/Library/Frameworks/Python.framework/Versions/2.5/Extras/lib/python  is being added to sys.path    import pkg_resources  test_shell_tests(43029 0xb0147000) malloc: *** error for object 0x14d943b0:  incorrect checksum for freed object - object was probably modified after  being freed.  *** set a breakpoint in malloc_error_break to debug  test_shell_tests(43029 0xb01c9000) malloc: *** error for object 0x14d960d0:  incorrect checksum for freed object - object was probably modified after  being freed.  *** set a breakpoint in malloc_error_break to debug  test_shell_tests(43029 0xb00c5000) malloc: *** error for object 0x14d960d0:  incorrect checksum for freed object - object was probably modified after  being freed.  *** set a breakpoint in malloc_error_break to debug  test_shell_tests(43029 0xb00c5000) malloc: *** error for object 0x14d960d0:  incorrect checksum for freed object - object was probably modified after  being freed.  *** set a breakpoint in malloc_error_break to debug  test_shell_tests(43029 0xb00c5000) malloc: *** error for object 0x14d960d0:  incorrect checksum for freed object - object was probably modified after  being freed.  *** set a breakpoint in malloc_error_break to debug  test_shell_tests(43029 0xb01c9000) malloc: *** error for object 0x14d960d0:  incorrect checksum for freed object - object was probably modified after  being freed.  *** set a breakpoint in malloc_error_break to debug  test_shell_tests(43029 0xb01c9000) malloc: *** error for object 0x14d960d0:  incorrect checksum for freed object - object was probably modified after  being freed.  *** set a breakpoint in malloc_error_break to debug  test_shell_tests(43029 0xb01c9000) malloc: *** error for object 0x14d960d0:  incorrect checksum for freed object - object was probably modified after  being freed.  *** set a breakpoint in malloc_error_break to debug    command timed out: 300 seconds without output  killing pid 43028  process killed by signal 9  program finished with exit code -1 </pre>